app/views/layouts/admin.html.erb

```diff
@@ -0,0 +1,41 @@
+<!doctype html>
+<html lang="en">
+  <head>
+    <!-- Required meta tags -->
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+
+    <!-- Bootstrap CSS -->
+    <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.1.3/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-1BmE4kWBq78iYhFldvKuhfTAU6auU8tT94WrHftjDbrCEXSU1oBoqyl2QvZ6jIW3" crossorigin="anonymous">
+
+    <title>Admin | Employee Recognition Platform</title>
+    
+    <meta name="viewport" content="width=device-width,initial-scale=1">
+    <%= csrf_meta_tags %>
+    <%= csp_meta_tag %>
+
+    <%= stylesheet_link_tag 'admin', media: 'all' %>
+    <%= javascript_pack_tag 'admin' %>
+    
+  </head>
+  <body>
+      <h1><%= link_to 'ADMIN | Employee Recognition', admin_root_path%></h1>
+      
+      <nav>
+          <h4>Navigation</h4>
+          <ul>
+             <% if !admin_user_signed_in? %>
+              <li><%= link_to 'Log in', new_admin_user_session_path %></li>
+             <% else %>
+              <li><%= link_to 'Log out', destroy_admin_user_session_path, method: :delete %></li>
+             <% end %>
+          </ul>
+          <hr />
+      </nav>   
```

Comment:
I would suggest moving admin navigation bar to a partial

Reply:
Yes, I will refine it, I wanted to meet the assumptions of the task first, I will refine the aesthetics as soon as possible :)